.github/issue_reporting_guide.md

```diff
@@ -0,0 +1,267 @@
+
+Sometimes the issue doesn’t reproduce every time. If you can put together a test case the only shows the problem 10-20% of times it is invoked it can still be very helpful for the maintainers.
+
+If you can’t put together a reproducible test case, you should still open an issue and try to provide as much other information as you can.
+
+## Describe your environment
+
+In addition to describing the problem make sure to provide enough details about the environment where you have confirmed the issue is appearing.
+
+There are lots of ways to run BuildKit, first make it clear which way you are using. Are you using `docker build` , a specific driver of `docker buildx` , running `buildkitd` or `buildctl` manually, in kubernetes, etc. 
+
+Sometimes it is not completely clear if the issue belongs in `moby/buildkit` , `docker/buildx` or even `moby/moby` repository. If you are unsure you can start from either BuildKit or Buildx and maintainers will fix it if needed. BuildKit is where your builds actually run, Buildx covers a Docker CLI experience and BuildKit deployment with drivers.
+
+Provide versions of the tools you used:
+
+- Using Buildkitd/Buildctl  `buildctl --version` `buildkitd version`
+- Using Docker Buildx `docker buildx version` and `docker buildx inspect` to return info about your current builder instance. If you run `docker build` then also report this.
+- If the issue is specific to Docker Engine embedded BuildKit or `docker buildx` Docker driver then report `docker version` and `docker info`
+
+If you are running an older version of BuildKit then ideally, you should try to reproduce the problem with the latest version before reporting it.
+
+The easiest way to test the latest BuildKit release is with the Docker Buildx container driver:
+
+```bash
+docker buildx create --name buildkit-latest --bootstrap
+export BUILDX_BUILDER=buildkit-latest
+docker buildx inspect
+```
+
+If you also want to check if the issue still appears in the `master` development branch you can run:
+
+```bash
+docker buildx build --load -t moby/buildkit:dev "https://github.com/moby/buildkit.git"
+docker buildx create --name buildkit-dev --driver-opt image=moby/buildkit:dev --bootstrap
+export BUILDX_BUILDER=buildkit-dev
+docker buildx inspect
+```
+
+To check if a specific existing PR is fixing the problem you can replace the first line with:
+
+```bash
+docker buildx build --load -t moby/buildkit:dev "https://github.com/moby/buildkit.git#pull/NR/head"
+```
+
+If your issue requires a specific environment or you have only encountered it in a specific environment then make sure to describe that.
+
+This would be for example specific BuildKit configurations:
+
+- Rootless mode
+- Non-default snapshotter
+- Special (insecure/mirror) registry configuration
+- Using containerd worker instead of default OCI one
+- Custom CNI configuration
+
+Or specific execution environment:
+
+- Docker Desktop or other Docker distro
+- AWS/EBS
```

Comment:
Does "EBS" here mean Elastic Block Store?

Reply:
(Not sure if Elastic Block Store is relevant here)

Reply:
I think we have issues atm with some (unconfirmed) reports of slowness where the EBS based storage seems to be the differentiating factor.